00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.02 Design Decisions/hub.yaml proposal for themes.md

```diff
@@ -0,0 +1,303 @@
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        print:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        callouts:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        checklists:
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        images: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        tables: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        asides: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        highlights: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        color_schemes: []
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        snippets: [] # Path to css snippets relative to the root of the repo. We could parse the css similar to style settings
+	        docs:
+                file: # from root of repo, to be included as is
+		        links: [] # markdown-formatted strings, e.g. pointing to the css files for download
+            screenshots: # Path to other screenshots from the root of the repo
+                - file: 
+                  caption:
+        css_classes: []
```

Comment:
just for the sake of consistency, I'd write `cssclass`, since the yaml-key in Obsidian also does not use any underscore. (I assume that some people might forget the underscore because they are used to write without it)

Reply:
Should be fixed in 524cc40